extension/js/common/core/mime.ts

```diff
@@ -156,7 +157,9 @@ export class Mime {
             } else if (Mime.getNodeType(node) === 'text/plain' && !Mime.getNodeFilename(node)) {
               mimeContent.text = Mime.getNodeContentAsUtfStr(node);
             } else if (Mime.getNodeType(node) === 'text/rfc822-headers') {
-              // todo - surface and render encrypted headers
+              if (node._parentNode && node._parentNode.headers.subject) {
```

Comment:
I'm not 100% sure here if `node._parentNode.headers.subject.length` should also be checked? Else you may get `undefined.value`

If you see other code that doesn't, than it's probably ok.

Reply:
yes, there is the same code above:
```TS
          for (const name of Object.keys(parser.node.headers)) {
            mimeContent.headers[name] = parser.node.headers[name][0].value;
          }
```

Reply:
Should I add `if` expression to both?

Reply:
I think not. If it worked well until now, it should work well the way you wrote it too. Thanks!